lib/otp_field.dart

```diff
@@ -47,23 +56,28 @@ class OTPTextField extends StatefulWidget {
   /// Callback function, called when pin is completed.
   final ValueChanged<String>? onCompleted;
 
-  OTPTextField(
-      {Key? key,
-      this.length = 4,
-      this.width = 10,
-      this.controller,
-      this.fieldWidth = 30,
-      this.margin: const EdgeInsets.symmetric(horizontal: 3),
```

Comment:
@JulianBissekkou here

Reply:
The margin was removed because it was tricky to find a good alignment.
And it was not possible to have padding between the text fields, but without the start and the end size.

So it doesn't align with the other elements in a list.

Reply:
what about people who may already be using this property, leave the margin as of now and mark it deprecated.

Reply:
Okay, added margin again and marked it as deprecated. Should work from now on :) @iamvivekkaushik